index.bs

```diff
@@ -263,6 +263,34 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">Support Existing Content</h3>
+
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing user agents.
+
+Content relying on existing user agent behavior can take many forms, such as:
+
+- Relying on features that are entirely proprietary.
+- Depending on specific error handling rules.
+- Depending on certain method names not being defined.
+- Depending on a feature from earlier specifications not being implemented as specified.
+
+When considering changes to legacy features, relative to current implementations and author expectations, the following questions should be considered:
```

Comment:
I think another question worth considering is something like "is the feature specified well enough and open enough that implementations that don't already implement it can do so".  (If the answer is no, I think that's an argument in favor of either (a) removal or (b) doing the work to change the answer to yes.)